Vagrantfile

```diff
@@ -8,4 +8,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "redash/dev"
   config.vm.synced_folder "./", "/opt/redash/current"
   config.vm.network "forwarded_port", guest: 5000, host: 9001
+  config.vm.provision "shell" do |s|
+    s.inline = "/opt/redash/current/install_requirements.sh"
```

Comment:
Let's use a single script (`vagrant_provision.sh`) that will also install the requirements. Also in the case of the Vagrant box, let's install all 3 files -- `requirements.txt`, `requirements_dev.txt` and `requirements_all_ds.txt` (this also means that there is no need for manual install of the PyMongo package).